osquery/CMakeLists.txt

```diff
@@ -291,40 +291,42 @@ if(NOT OSQUERY_BUILD_SDK_ONLY)
     set_target_properties(libosquery_additional_shared PROPERTIES OUTPUT_NAME osquery_additional)
   endif()
 
-  add_executable(shell devtools/shell.cpp main/shell.cpp)
-  ADD_DEFAULT_LINKS(shell TRUE)
-  SET_OSQUERY_COMPILE(shell "${CXX_COMPILE_FLAGS}")
-  target_link_libraries(shell)
-  set_target_properties(shell PROPERTIES OUTPUT_NAME osqueryi)
-
-  add_executable(daemon main/${PROCESS_FAMILY}/daemon.cpp)
+  add_executable(daemon
+    devtools/shell.cpp
+    main/main.cpp
+    main/${PROCESS_FAMILY}/main.cpp
+  )
   ADD_DEFAULT_LINKS(daemon TRUE)
   SET_OSQUERY_COMPILE(daemon "${CXX_COMPILE_FLAGS}")
   set_target_properties(daemon PROPERTIES OUTPUT_NAME osqueryd)
 
-  # Include the public API includes if make devel.
-  install(TARGETS libosquery ARCHIVE DESTINATION lib COMPONENT devel OPTIONAL)
-  install(DIRECTORY "${CMAKE_SOURCE_DIR}/include/osquery"
-    DESTINATION include
-    COMPONENT devel OPTIONAL
-    PATTERN ".*" EXCLUDE
-  )
-
-  # A friendly echo printed before building the shell
-  add_custom_command(TARGET shell PRE_BUILD
-    COMMAND ${CMAKE_COMMAND} -E cmake_echo_color --cyan --bold
-      "-- Building osqueryi: $<TARGET_FILE:shell>"
-  )
-
   # A friendly echo printed before building the daemon
   add_custom_command(TARGET daemon PRE_BUILD
     COMMAND ${CMAKE_COMMAND} -E cmake_echo_color --cyan --bold
       "-- Building osqueryd: $<TARGET_FILE:daemon>"
   )
 
+  add_custom_target(shell ALL
```

Comment:
Is the output `osqueryi` binary on Windows created as `osqueryi.exe` in the PR? Event though CMake is writing `osqueryi` without the `.exe`?